docs/source/index.rst

```diff
@@ -77,10 +64,17 @@ Users of the ingest system will likely fall into 2 categories:
 
 2. **Ingest Package Developer**
 
-   - Runs ingest packages
```

Comment:
Doesn't this statement still need to be here? In most cases (and not in the near future), an ingest package developer would always be a ingest operator as well. Then, "Will never create a new study ingest package" seems conflicting the roles of an ingest package developer.

Reply:
I think a person can have multiple roles.

Reply:
> Then, "Will never create a new study ingest package" seems conflicting the roles of an ingest package developer.

Yeah, I'll remove it.

> Doesn't this statement still need to be here? In most cases (and not in the near future), an ingest package developer would always be a ingest operator as well. 

I think removing the above makes it safe to say that people can have multiple roles.